src/SFA.DAS.ApprenticeCommitments.Web.AcceptanceTests/Steps/ConfirmIdentitySteps.cs

```diff
@@ -97,5 +104,90 @@ public void WhenTheApprenticeShouldBeShownThePage(string page)
+                })
+            };
+
+           await _context.Web.Send(request);
+
+        }
+
+
+        //[When(@"the apprentice verifies their identity with")]
+        //public async Task GivenTheApprenticeVerifiesTheirIdentityWith(Table table)
+        //{
+        //    _context.OuterApi.MockServer
+        //        .Given(Request.Create().WithPath("/registrations*"))
+        //        .RespondWith(Response.Create().WithStatusCode(HttpStatusCode.OK));
+
+        //    _postedRegistration = table.CreateInstance(() => new ConfirmYourIdentityModel(null));
+
+        //    var get = await _context.Web.Get("ConfirmYourIdentity");
+        //    using var content = await HtmlHelpers.GetDocumentAsync(get);
+
+        //    var form = (IHtmlFormElement)content.QuerySelector("form");
+        //    (form["FirstName"] as IHtmlInputElement).Value = _postedRegistration.FirstName;
+        //    (form["LastName"] as IHtmlInputElement).Value = _postedRegistration.LastName;
+        //    (form["NationalInsuranceNumber"] as IHtmlInputElement).Value = _postedRegistration.NationalInsuranceNumber;
+
+        //    var button = (IHtmlButtonElement)content.QuerySelector("button");
+        //    var formSubmission = form.GetSubmission(button);
+
+        //    var target = (Uri)formSubmission.Target;
+
+        //    var submitted = new StreamReader(formSubmission.Body).ReadToEnd();
+
+        //    var request = new HttpRequestMessage(new HttpMethod(formSubmission.Method.ToString()), target)
+        //    {
+        //        Content = new StreamContent(formSubmission.Body)
+        //    };
+
+        //    request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/x-www-form-urlencoded");
+
+        //    await _context.Web.Client.SendAsync(request);
+        //}
+
+        [Then("verification is successfull")]
+        public void ThenTheVerificationIsSuccessfull()
+        {
+            ThenTheResponseStatusCodeShouldBeOk();
+            var registrationPosts = _context.OuterApi.MockServer.FindLogEntries(
+                Request.Create()
+                    .WithPath($"/registrations*")
+                    .UsingPost()
+                                                                               );
+
+            registrationPosts.Should().NotBeEmpty();
+
+            var post = registrationPosts.First();
+
+            post.RequestMessage.Path.Should().Be($"/registrations/{_registrationId}");
+            var reg = JsonConvert.DeserializeObject<VerifyRegistrationCommand>(post.RequestMessage.Body);
+            reg.Should().BeEquivalentTo(new VerifyRegistrationCommand
+            {
+                RegistrationId = _registrationId,
+                FirstName = _postedRegistration.FirstName,
+                LastName = _postedRegistration.LastName,
+                NationalInsuranceNumber = _postedRegistration.NationalInsuranceNumber,
+            });
+        }
```

Comment:
I believe this should be split into two assertions, one that we got a certain  IActionResult back and teh second that we posted to the VerfiyRegistrations endpoint correctly,